rpc/gnmi/gnmi-specification.md

```diff
@@ -453,6 +453,34 @@ update: <
 >
 ```
 
+For `/a/b[name=b1]`:
+
+```
+update: <
+  path: <
+    elem: <
+      name: "a"
+    >
+    elem: <
+      name: "b"
+      key: <
+        key: "name"
+        value: "b1"
+      >
+    >
+  >
+  val: <
+    json_ietf_val: `{ 
+                      "name": "b1",
+                      "c": {
+                        "d": "AStringValue",
+                        "e": 10042
+                      }
+            }`
+  >
+>
+```
+
```

Comment:
Is it worth explicitly stating here that the contents of the update is the JSON object that would be expected at that list entry?